library/src/main/java/com/novoda/downloadmanager/DownloadBatchStatus.java

```diff
@@ -33,27 +39,61 @@ public static Status from(String rawValue) {
 
     }
 
+    /**
+     * @return The title associated to a {@link DownloadBatch}.
+     * Specified when calling {@link Batch#with(StorageRoot, DownloadBatchId, String)}.
```

Comment:
Maybe we can add that this title is going to be used by the default notifications